packages/core/src/invoke.ts

```diff
@@ -80,3 +94,53 @@ export function invokeObservable<T extends EventObject = AnyEventObject>(
     return createObservableBehavior(resolvedSource, parent);
   };
 }
+
+export function createInvocation<TContext, TEvent extends EventObject>(
```

Comment:
This is not new logic; it was moved from `interpreter.ts`, everything is still the same.

Reply:
this name is slightly confusing to me as this returns an actor ref and "invocation" sounds like a new/different concept

Reply:
Renamed to... something slightly more descriptive